test/playlist-maker.test.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable function-paren-newline */
```

Comment:
videojs-standard update caused failures for this particular rule

Reply:
does `vjsstandard --fix` fix this rule? If so, it's fine to make the change here as a separate commit